README.md

```diff
@@ -1,28 +1,51 @@
-# Optimistic specs
-
-Shared open-source specification for an optimistic-rollup that strives for 1:1 Ethereum on L2 while minimizing additional code.
-
-The dream: *EVM compatibility, all Execution clients, all L1 tooling, L2 security (rollup), no L1-like limits*
-
-This project is a combined effort of the Optimism, EF research and Quilt/Consensys research teams.
-
-Enabling rollups on execution-layer data today, and shard-data in the future.
-
-[**Specs overview**](./overview.md) - experimental!
-
-Components (under construction):
-- [Layer 1 Contracts](./components/layer1.md)
-- [Rollup Client](./components/rollup_client.md)
-- [Execution Engine](./components/exec_engine.md)
-- [Batch Submitter](./components/batch_submitter.md)
-- [Witness Generator](./components/witness_gen.md)
-- [Challenge Agent](./components/challenge_agent.md)
-
-## Contribute
-
-Contribute by opening a PR. There are weekly dev calls you can join, 
-chat with [@protolambda](https://github.com/protolambda/) or [@karlfloersch](https://twitter.com/karl_dot_tech/).
-
+# The Optimistic Ethereum Spec
+
+This repository holds the open-source specification for Optimistic Ethereum, an ultra-minimal optimistic rollup protocol that maintains 1:1 compatibility with Ethereum.
+
+## Work in Progress
+Please note that this specification is currently heavily under construction.
+You will find that several pages are incomplete or [under discussion](https://github.com/ethereum-optimism/optimistic-specs/discussions).
+
+## About the project
+
+Early designs for Optimistic Ethereum were spearheaded by [Optimism](https://optimism.io/), which actively maintains an implementation of the protocol at the [optimism monorepo](https://github.com/ethereum-optimism/optimism).
+Recent versions of the protocol have been vastly simplified to the point that we can now envision a future in which Optimistic Ethereum can act as a standardized optimistic rollup design.
```

Comment:
Perhaps we could reword this sentence to be something like "we are working towards developing standards..." instead of saying "we should make optimism the standard"